lib/srv/desktop/rdp/rdpclient/src/lib.rs

```diff
@@ -1128,27 +1129,47 @@ pub unsafe extern "C" fn handle_tdp_sd_move_response(
 /// `read_rdp_output` reads incoming RDP bitmap frames from client at client_ref and forwards them to
 /// handle_bitmap.
 ///
+/// The ErrCodeAndMessage value that's returned should be interpreted as:
+///
+/// The function ended due to a client disconnect
+/// ErrCodeAndMessage {
+///     message: "",
+///     err: CGOErrCode::ErrCodeSuccess,
```

Comment:
Could we have unambiguous codes for client and server disconnects? As soon as we add a message to client disconnects this scheme breaks.

Reply:
Yeah, I'll add a `DisconnectCode`

Reply:
2598c7f3626e74013dd2b5fed943853db8691bd3